airflow/config_templates/config.yml

```diff
@@ -402,9 +402,9 @@
       type: string
       example: ~
       default: ""
-    - name: stackdriver_key_path
```

Comment:
I am not sure if there is 'deprecation policy' but if someone was actually using this setting. After upgrade it will stop working.

Reply:
This option has not yet been found in any release.